### DISCUSSION TITLE
Practical Series 4 - Breakout Room 3

### DISCUSSION THREAD

**Discussion by 2199105T** [IBAHCM/RPiR#269] [Archive]
Sarah Tonnar @2199105T

Kelvin Munisi @KelvinMunisi92

Dominik Vinopal  @DominikVinopal

Chuqiao Li @Chuqiaoli

**Comment by KelvinMunisi92** [comment]
hey guys..! @2199105T @DominikVinopal @chuqiaoli may you please check my code if it runs without an error. 
And for @DominikVinopal  and @chuqiaoli, may I have access to your code.
Thanks,
Kelvin

**Reply by chuqiaoli** [reply]
yes sure. I have invited you to access my repository, please have a check.

**Reply by DominikVinopal** [reply]
I was able to install it and run the demo without any problems.  I also gave you access to my repo so if you could check. The description is not finished but it should run

**Reply by 2199105T** [reply]
Hi @KelvinMunisi92,

I managed to install your package and run the demo with no errors :)

Do you mind trying to install mine?

Sarah

**Reply by chuqiaoli** [reply]
@KelvinMunisi92 
I have installed your package and run demos, just a reminder to delete the hello.R in R file. 
good job!!

**Comment by chuqiaoli** [comment]
https://github.com/IBAHCM/chuqiaoliBCI
